roles/foreman_installer_devel_scenario/templates/katello-devel-answers.yaml

```diff
@@ -18,16 +19,9 @@ foreman_proxy:
   foreman_ssl_ca: /etc/foreman-proxy/foreman_ssl_ca.pem
   foreman_ssl_cert: /etc/foreman-proxy/foreman_ssl_cert.pem
   foreman_ssl_key: /etc/foreman-proxy/foreman_ssl_key.pem
-puppet:
-  server: true
-  server_environments_owner: apache
-  server_foreman_ssl_ca: /etc/pki/katello/puppet/puppet_client_ca.crt
-  server_foreman_ssl_cert: /etc/pki/katello/puppet/puppet_client.crt
-  server_foreman_ssl_key: /etc/pki/katello/puppet/puppet_client.key
-  # Tune the server for a minimal deployment
-  puppet_server_jvm_max_heap_size: 1G
-  puppet_server_jvm_min_heap_size: 1G
-  puppet_server_max_active_instances: 1
+  puppet: false
+  puppetca: false
```

Comment:
I think this needs `manage_puppet_group: false` here.

```
    2021-07-21 19:29:15 [ERROR ] [configure] Evaluation Error: Error while evaluating a Resource Statement, Evaluation Error: Error while evaluating a Resource Statement, Duplicate declaration: File[/etc/foreman-proxy/ssl_key.pem] is already declared at (file: /usr/share/foreman-installer/modules/foreman_proxy/manifests/config.pp, line: 135); cannot redeclare (file: /usr/share/foreman-installer/modules/certs/manifests/keypair.pp, line: 36) (file: /usr/share/foreman-installer/modules/certs/manifests/keypair.pp, line: 36, column: 5) (file: /usr/share/foreman-installer/modules/certs/manifests/foreman_proxy.pp, line: 95) on node centos7-devel.wareagle.example.com
```

The turning off of puppet features triggers this block https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/config.pp#L118

Perhaps that code could use an update to be smarter about determining management of the puppet group.

Reply:
Ah yes, we should. Then we can also get rid of https://github.com/theforeman/foreman-installer/blob/bc575c2d662dad4779623b89d24ca7750dcec3bd/config/katello-answers.yaml#L73 and https://github.com/theforeman/foreman-installer/blob/bc575c2d662dad4779623b89d24ca7750dcec3bd/config/foreman-proxy-content-answers.yaml#L21. For now I'll add it here too.

Reply:
Is this something you could capture in an installer issue? I think you'll word it better than I will.